### DISCUSSION TITLE
Android tv

### DISCUSSION THREAD

**Discussion by Sergey5588** [dovecoteescapee/ByeDPIAndroid#183] [Q&A]
Работает на андроид ТВ или не?

**Comment by Sergey5588** [comment]
Vpn режим не работает

**Comment by thirstyone** [comment]
в смыле в android tv не поддерживается vpn режим? или на конкретной модели телевизора залочена такая возможность?  
а само приложение-то можно запустить? (тоже намерен, когда доберусь, попробовать на xiaomi tv stick)
